### PR TITLE
feat(open-mysql-db): open mysql db cicd

### PR DESCRIPTION
**Pull request by yangwucheng**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
enable open mysql db cicd

### CODE REVIEW

File: .github/workflows/open-mysql-db.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: OpenM(ysq)LDB_CICD
+
+on:
+  push:
```
Comment (aceforeverd) [inline .github/workflows/open-mysql-db.yml:4]: use the `paths` attribute, only trigger the job if PR changes relevant files

Comment (tobegit3hub) [reply .github/workflows/open-mysql-db.yml:4]: Refer to the cicd yaml files in openmldb. May be "pull_request".

Comment (yangwucheng) [reply .github/workflows/open-mysql-db.yml:4]: DONE

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**